### ISSUE TITLE
Add ability to clear "to" addresses

### ISSUE THREAD

**Issue by coffeeshopped** [Znarkus/postmark-php#6]
It would be useful to have a method named clearRecipients() or something that would allow for the recipients (to, cc, bcc) to be cleared out.  That way a single instance of the postmark class could be reused to send an email to a few people without them all being in the same email.

**Comment by Znarkus** [comment]
Agree. But until I implement it, you could use ::to() which will will clear the "to" recipients.

https://github.com/Znarkus/postmark-php/blob/master/Postmark.php#L415

But I can't really recommend that, because the to() method is deprecated.
